static/src/server.ts

```diff
@@ -342,78 +502,49 @@ app.get("/nodejs/query", (req: Request, res: Response) => {
           enclosedPlaceType;
       }
 
-      // Get the list of tiles and the stat var spec for the first category in
-      // the config.
-      const categories = config["categories"];
-      if (!_.isEmpty(categories)) {
-        const category = categories[0];
+      // If no place, return here
+      if (!place.dcid) {
+        res.status(200).send("No data found");
```

Comment:
Since response is a json, should this be a structured object. Like have an "err" field in the response and set it here.

Reply:
Updated so that the 200 responses with no data just return an empty array for charts & the 500 responses return a json object with an "err" field